Remotion/ObjectBinding/Web/Res/HTML/BocAutoCompleteReferenceValue.jquery.js

```diff
@@ -1445,7 +1445,10 @@
         };
     };
 
-    $.Autocompleter.Selection = function(field, start, end) {
+    $.Autocompleter.Selection = function(field, start, end, focusInputAfterSelection) {
+        if (field !== document.activeElement && !focusInputAfterSelection)
+            return;
+

```

Comment:
This condition would only be needed in 1.20 due to the existing IE support.

Reply:
yeah, atm I'd just do IE. we will make the cut and cleanup in January when we setup a feature-branch-support for 1.20 in CI